webapp/src/components/assignee_modal/assignee_modal.jsx

```diff
@@ -45,6 +45,7 @@ const AssigneeModal = (
         } else {
             removeAssignee();
         }
+        setAssignee(null);
```

Comment:
@asaadmahmood I'm curious why we used a `div` instead of Bootstrap's `Modal` component here. Not really any issues with that, just deviating from the pattern of other plugins using the Bootstrap modal